src/components/ArticleForm/ArticleForm.jsx

```diff
@@ -6,13 +6,20 @@ import { toast } from 'react-toastify';
 const ArticleForm = ({ onSubmit, loading, countryUrl, continentUrl }) => {
     const [formErrors, setFormErrors] = useState({});
     const today = new Date();
-    today.setDate(today.getDate() - 1);
+
     const [form, setForm] = useState({
         country: '',
-        date: today.toISOString().split('T')[0],
+        date: getStartDayOftheWeek(today),
         access: 'all-access',
     });
 
+    function getStartDayOftheWeek(date) {
+        const diff = date.getDate() - date.getDay() + (date.getDay() === 0 ? -6 : 1);
```

Comment:
@ShadowFax1731 can you explain the logic here ?

I can see that you are trying to handle it manually, try with the [momentjs](https://momentjs.com/) package that already handles the first weekday for different locales:

```javascript
    const firstDayOfWeek = moment().startOf('week').format('YYYY-MM-DD');
```